sam/bamSeek_test.go

```diff
@@ -41,6 +41,48 @@ func TestSeekBamRegion(t *testing.T) {
 	}
 }
 
+func TestSeekBamRegionRecycle(t *testing.T) {
+	br, _ := OpenBam("testdata/rand.bam")
+	bai := ReadBai("testdata/rand.bam.bai")
+	reads := make([]Sam, 10)
+	reads = SeekBamRegionRecycle(br, bai, "chr7", 45000000, 45200000, reads)
+	for i := range reads {
+		if reads[i].RName != "chr7" || !(reads[i].GetChromStart() < 45200000 && reads[i].GetChromEnd() > 45000000) {
+			t.Error("problem with SeekBamRegion")
```

Comment:
Would the error message in this function be "problem with SeekBamRegionRecycle"? As far as I understand, since you test "SeekBamRegion" above, if only this function fails, you'll know it's the non-nil recycledSams []Sam that caused it to fail?

Reply:
Good catch. Fixed.